packages/network-controller/src/NetworkController.ts

```diff
@@ -1233,60 +1232,41 @@ export class NetworkController extends BaseController<
         this.state.networkConfigurationsByChainId,
       );
 
-    this.messenger.registerActionHandler(
-      // TODO: Either fix this lint violation or explain why it's necessary to ignore.
-      // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
-      `${this.name}:getEthQuery`,
-      () => {
-        return this.#ethQuery;
-      },
-    );
+    this.messenger.registerActionHandler(`${this.name}:getEthQuery`, () => {
+      return this.#ethQuery;
+    });
 
     this.messenger.registerActionHandler(
-      // TODO: Either fix this lint violation or explain why it's necessary to ignore.
-      // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
```

Comment:
Why are all of these no longer required? 🤔

Reply:
When these lint violations appeared, at the time we were using an older version of the `typescript-eslint` packages. When we upgraded, they went away.